features/step_definitions/assertion_steps.rb

```diff
@@ -44,6 +54,22 @@
   expect(current_path).to eq new_user_password_path
 end
 
+Then("the average rating for {string} should be {string}") do |recipe_title, count|
+  recipe = Recipe.find_by(title: recipe_title)
+  # TODO: calculate average and save in recipe.rating attribute
+  # Should probably be renamed to average_rating
+
+  # total = recipe.ratings.map(&:value).sum
+  # instances = recipe.ratings.any? ? recipe.ratings.count : 1  #to avoid division by zero error
+  # average = total / instances
+  # recipe.update_attribute(:rating, average)
+  expect(recipe.rating).to eq count.to_i
```

Comment:
We want to test the `average_rating` here, right?  `count` is no longer a valid variable name. Perhaps ` value`?